sql/mssql/00020-Evaluation-Tables.sql

```diff
@@ -0,0 +1,149 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE eppeta.PerformanceEvaluation (
+    EducationOrganizationId INT NOT NULL,
+    EvaluationPeriodDescriptorId INT NOT NULL,
+    PerformanceEvaluationTitle VARCHAR(50) NOT NULL,
+    PerformanceEvaluationTypeDescriptorId INT NOT NULL,
+    SchoolYear SMALLINT NOT NULL,
+    TermDescriptorId INT NOT NULL,
+    PerformanceEvaluationDescription VARCHAR(255) NULL,
+    CreateDate DATETIME2 NOT NULL,
+    LastModifiedDate DATETIME2 NOT NULL,
+    Ods_Id UNIQUEIDENTIFIER NULL,
```

Comment:
Suggestion: rename to `EdFi_Id` and change to a string. "ODS" is the name of a database, not an API. So we want to make this a little more generic. Normally we don't use underscores in SQL Server naming conventions, but I like your use of it here, especially when changing the name: `EdFiId` is hard to read 😆 .

Also change to a string so that we can accommodate other versions of an Ed-Fi API that use strings but not necessarily UUID values. `varchar(50)` seems fine.